tasks/variables.js

```diff
@@ -1,12 +1,18 @@
+var fs = require( "fs" );
 module.exports = function( grunt ) {
 	grunt.registerTask( "variables", function() {
 		var jsass = require( "jsass-vars" );
 		var path = require( "path" );
 		var varPath = path.join( process.cwd(), "scss/variables/*" );
-
-		jsass( varPath, {
-			"dest": path.join( process.cwd(), "dist" ),
-			"name": "chassis"
-		} );
+		var distPath = path.join( process.cwd(), "dist" );
+		fs.stat( distPath, function( err, stat ) {
```

Comment:
no need to check if the path already exists just do `grunt.file.mkdir( distPath );`